Makefile

```diff
@@ -58,22 +62,30 @@ secretkey:
 	openssl rand -hex 24
 
 jetstream_config:
-	curl -LJ -o app/experimenter/outcomes/jetstream-config.zip https://github.com/mozilla/jetstream-config/archive/main.zip
+	curl -LJ -o app/experimenter/outcomes/jetstream-config.zip $(JETSTREAM_CONFIG_URL)
 	unzip -o -d app/experimenter/outcomes app/experimenter/outcomes/jetstream-config.zip
 
+feature_manifests:
+	curl -LJ --create-dirs -o app/experimenter/features/manifests/firefox-desktop.js $(FEATURE_MANIFEST_DESKTOP_URL)
+	echo "console.log(JSON.stringify(FeatureManifest));" >> app/experimenter/features/manifests/firefox-desktop.js
```

Comment:
Well that's a nifty quick & dirty JSON conversion. Might be worth wrapping this up in a shell script? But it's only 2 lines, so also probably not worth thinking about until/unless it grows.

Might also be worth running inside a container in case the host doesn't have node? Also probably not a practical concern.

Reply:
Yeah I know I was thinking the same but we don't currently have an entry point to stick it... and I wanted to keep it all together.  And yeah tacking console.log on to the end is super hacky but the js files in mc are weird and I couldn't get it to be importable by a separate script. 

And yes I just realized I should update the README to indicate installing node as a build requirement which is only listed under the semi dockerized setup.